Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -5,10 +5,10 @@ import Foundation
 import CoreLocation
 import MapboxCommon
 import MapboxNavigationNative
-import MapboxMobileEvents
 import MapboxDirections
 import Polyline
 import Turf
+import UIKit
```

Comment:
And here?

Reply:
`UIDevice` reference. `UIKit` was imported transitively through MME.